usaspending_api/api_contracts/contracts/v2/idvs/count/federal_account.md

```diff
@@ -0,0 +1,27 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Federal Accounts Count [/api/v2/idvs/count/federal_account/{award_id}/]
```

Comment:
Can you change this file name to be `/api_contracts/contracts/v2/idvs/count/federal_account/award_id.md` since the endpoint requires the `award_id`? This came up with `/api_contracts/contracts/v2/federal_accounts/account_number.md` and `/api_contracts/contracts/v2/federal_accounts` as well.

Reply:
Also, if something such as `/api/v2/idvs/count/federal_account/2/?piid=piid_011` is allowed then the contract should reflect the query parameter.